test/client/setup-test-framework.js

```diff
@@ -23,6 +28,8 @@ jest.mock( 'enzyme', () => {
 			chai.use( chaiEnzyme() );
 		}
 	}
+	// configure enzyme for React 16, from docs: http://airbnb.io/enzyme/docs/installation/index.html
+	Adapter.apply && actualEnzyme.configure( { adapter: new Adapter() } );
```

Comment:
It should be guarded with `if ( ! mockEnzymeSetup ) {` as far as I remember.

Reply:
I refactored this lazy loading logic with https://github.com/Automattic/wp-calypso/pull/19083/commits/e5944a5341314ab06bc36911e33138ecdcdf319e. I think it was too complicated :D